core/client/app/components/gh-post-tags-input.js

```diff
@@ -0,0 +1,149 @@
+    //             isComma = ','.localeCompare(String.fromCharCode(event.keyCode || event.charCode)) === 0;
+
+    //         // use localeCompare in case of international keyboard layout
+    //         if (isComma) {
+    //             event.preventDefault();
+
+    //             if (controller.get('selectedSuggestion')) {
+    //                 controller.send('addSelectedSuggestion');
+    //             } else {
+    //                 controller.send('addNewTag');
+    //             }
+    //         }
+    //     },
+
+    //     keyDown: function (event) {
+    //         var controller = this.get('parentView.controller'),
+    //             keys = this.get('parentView.keys'),
+    //             hasValue;
+
+    //         switch (event.keyCode) {
+    //             case keys.UP:
+    //                 event.preventDefault();
+    //                 controller.send('selectPreviousSuggestion');
+    //                 break;
+
+    //             case keys.DOWN:
+    //                 event.preventDefault();
+    //                 controller.send('selectNextSuggestion');
+    //                 break;
+
+    //             case keys.TAB:
+    //             case keys.ENTER:
+    //             case keys.NUMPAD_ENTER:
+    //                 if (controller.get('selectedSuggestion')) {
+    //                     event.preventDefault();
+    //                     controller.send('addSelectedSuggestion');
+    //                 } else {
+    //                     // allow user to tab out of field if input is empty
+    //                     hasValue = !Ember.isEmpty(this.get('value'));
+    //                     if (hasValue || event.keyCode !== keys.TAB) {
+    //                         event.preventDefault();
+    //                         controller.send('addNewTag');
+    //                     }
+    //                 }
+    //                 break;
+
+    //             case keys.BACKSPACE:
+    //                 if (Ember.isEmpty(this.get('value'))) {
+    //                     event.preventDefault();
+    //                     controller.send('deleteLastTag');
+    //                 }
+    //                 break;
+
+    //             case keys.ESCAPE:
+    //                 event.preventDefault();
+    //                 controller.send('reset');
+    //                 break;
+    //         }
+    //     }
+    // }),
+
+    // suggestionView: Ember.View.extend({
+    //     tagName: 'li',
+    //     classNameBindings: 'suggestion.selected',
+
+    //     suggestion: null,
+
+    //     // we can't use the 'click' event here as the focusOut event on the
+    //     // input will fire first
+
+    //     mouseDown: function (event) {
+    //         event.preventDefault();
+    //     },
+
+    //     mouseUp: function (event) {
+    //         event.preventDefault();
+    //         this.get('parentView.controller').send('addTag',
+    //             this.get('suggestion.tag'));
+    //     }
+    // }),
```

Comment:
Eventually (soon).  The tag input widget is going to be reimplemented, so I did enough to clear the deprecation warnings on the existing implementation and then left it there as a reference.